docs/scripts/trans-copy

```diff
@@ -28,6 +28,8 @@ for SOURCE in $(find ${ROOT}/content -type f -iname *.en-us.md); do
     if [[ ! -f ${DEST} ]]; then
       echo "Creating fallback for ${DEST#${ROOT}/content/}"
       cp ${SOURCE} ${DEST}
+      sed -i.bak "s/en\-us/${LOCALE}/g" ${DEST}
+      rm ${DEST}.bak
```

Comment:
why not a simple `sed -i` without `.bak` so that it modifies the file in place?

Reply:
Because on macosx sometimes it has problems by directly replacing in same file